core/include/traccc/finding/actors/ckf_aborter.hpp

```diff
@@ -61,6 +58,16 @@ struct ckf_aborter : detray::actor {
                 "CKF: Maximum number of steps to reach next sensitive surface "
                 "exceeded");
         }
+
+        const auto &track = stepping();
+        const scalar q{stepping.particle_hypothesis().charge()};
+        const scalar mag{track.pT(q)};
+
+        if (mag <= 500.f * traccc::unit<scalar>::MeV) {
```

Comment:
Spotted this while randomly partolling the open PRs.
We should not abort the navigator just because its momentum is low

 Could you please find another way to handle low momentum? For example, I can suggest to add a configuration parameter (min_pT) to  finding config, propagation config or fitting config. Of course, the default value should be much lower (e.g. 0) - 500 MeV/c is unacceptably high.